test/unit/storage_subformat_test.py

```diff
@@ -112,22 +112,29 @@ def test_disk_format_ova(self, mock_ova):
             {'adapter_type=controller': None, 'subformat=disk-mode': None}
         )
 
+    @patch('kiwi.storage.subformat.DiskFormatVagrantVirtualBox')
```

Comment:
Why are we patching VBox stuff for a libvirt test?

Reply:
I think that this function is testing the `__init__.py` from `storage/subformat/`, more specifically the `elif name == "vagrant"` branch in `DiskFormat.__new__`. The mock is in place so that we get no side effects from actually constructing a `DiskFormatVagrantVirtualBox` or a `DiskFormatVagrantLibVirt`. I have added the additional mock, since the test for the construction of a  `DiskFormatVagrantVirtualBox` would be just a duplicate of the existing test for libvirt.